### DISCUSSION TITLE
Add onClear prop to MultiSelect

### DISCUSSION THREAD

**Discussion by baba43** [mantinedev/mantine#1152] [Feature requests]
We would like to remove the focus of the input field when a user "clears" the field, since we use this as a filter.

For this, it would be enough if we could catch the "onClear" action. This is currently not possible if I see correctly, or is there another way for this?

Thank you very much at this point for this incredibly good project.

**Comment by rtivital** [comment]
You can detect that Select was cleared when `onChange` is called with null.

**Reply by jack-szeto** [reply]
Yes but it would be easier if we could have an `onClear` callback to get the previous selected value

**Comment by aman1205** [comment]
When I call onClear it set my state to empty , bbuut I want to set the state to  give value 
OnClear={()=>setYears(['2023','2022'])} in multiple select , above mention function is working it set empty array
